### PR TITLE
Create Meeting dates and times

### PR DESCRIPTION
**Pull request by pierluigividale**

### CODE REVIEW

File: Meeting dates and times [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Meeting dates:
+
++ 18 Nov 2024
++ 16 Dec 2024
```
Comment (florianziemen) [inline Meeting dates and times:4]:
```suggestion
+ 19 Dec 2024
```

File: Meeting dates and times [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
++ 9  Jan 2025
++ 9  Feb 2025
```
Comment (florianziemen) [inline Meeting dates and times:6]:
```suggestion
+ 11  Feb 2025
```

File: Meeting dates and times [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Meeting dates:
```
Comment (florianziemen) [inline Meeting dates and times:1]:
```suggestion
# Preparation meetings
 
## Meeting dates
```

**Review by florianziemen — COMMENTED** [review]
We'll have to rename it to .md and put it somewhere sane (maybe simply in Hosting/?), but let's first sort out the actual dates...

**Comment by pierluigividale** [comment]
> We'll have to rename it to .md and put it somewhere sane (maybe simply in Hosting/?), but let's first sort out the actual dates...

Ah! That is why I could not render it...

**Review by florianziemen — APPROVED** [review]
We should probably include some info how to get the zoom link.
Otherwise fine with me now.

**Pull request merged**